keras_cv/models/stable_diffusion/stable_diffusion_test.py

```diff
@@ -13,24 +13,23 @@
 # limitations under the License.
 
 import pytest
-import tensorflow as tf
 from tensorflow.keras import mixed_precision
 
+from keras_cv.backend import ops
+from keras_cv.backend import random
 from keras_cv.models import StableDiffusion
 from keras_cv.tests.test_case import TestCase
 
 
 @pytest.mark.extra_large
-@pytest.mark.tf_keras_only
 class StableDiffusionTest(TestCase):
+    @pytest.mark.large
```

Comment:
Actually this is already marked for GCB testing. Don't remember if I did this or you, but perfect!